pages/cadastro/index.public.js

```diff
@@ -172,13 +184,21 @@ function SignUpForm() {
           setErrorObject={setErrorObject}
         />
 
+        <Box sx={{ display: 'flex', alignItems: 'center', gap: 3 }}>
+          <input type="checkbox" checked={isCheckboxChecked} onClick={() => setIsCheckboxChecked(!isCheckboxChecked)} />
```

Comment:
Melhor usar o [`Checkbox` da biblioteca Primer](https://primer.style/react/Checkbox), pois mesmo ele ainda estando em versão alpha, conforme a biblioteca for evoluindo, vamos ganhando as melhorias, por exemplo, de estilização e acessibilidade.

Se tiver problemas com alinhamento, talvez o que foi feito no PR #1173 possa ajudar.